tests/x11regressions/empathy/empathy_irc.pm

```diff
@@ -48,16 +48,13 @@ sub run() {
     };
 
     # join openqa channel
-    if (sle_version_at_least('12-SP2')) {
+    send_key "ctrl-j";
+    if (!check_screen 'empathy-join-room', 4) {
```

Comment:
not nice but it would be ok in this special case just to be done with it :-)
@kravciak is right that `wait_screen_change` will react on anything changing on the screen so it could be that the 'ctrl-j' does not have any effect but the clock changes and `wait_screen_change` exits. Anyway, it would probably work good enough because the `wait_screen_change` at least looked at the screen once and during that time the menu should open so I still think my approach should work.